jni_gen/lib/src/writers/files_writer.dart

```diff
@@ -0,0 +1,126 @@
+// Copyright (c) 2022, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:io';
+
+import 'package:jni_gen/src/bindings/bindings.dart';
+
+import 'package:jni_gen/src/elements/elements.dart';
+import 'package:jni_gen/src/config/wrapper_options.dart';
+import 'package:jni_gen/src/util/find_package.dart';
+
+import 'bindings_writer.dart';
+
+// Writes dart files to specified directory
+// The name of the files themselves is decided by package name
+class FilesWriter extends BindingsWriter {
```

Comment:
Maybe

/// Writer of generated dart files. 
///
/// Writes dart files to specified directory.
/// The name of the files themselves is decided by package name.